src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -149,6 +152,11 @@ export class MdAutocompleteTrigger implements AfterContentInit, ControlValueAcce
     return this._keyManager.activeItem as MdOption;
   }
 
+  /** The initial list of autocomplete options, as soon as the zone has stabilized */
+  get initialOptionList(): Observable<QueryList<MdOption>> {
```

Comment:
I don't think this is really the initial list of options- it's just whatever the next list of options are once the zone stabilizes; the fact that you call it on initialization is separate from what you're getting. So you could change this to `_getStableOptions` and then in the subscribe function do
```ts
let initialOptions = this._getStableOptions();`
```

Also should be private or internal.